content/en/service_catalog/manage_entries.md

```diff
@@ -110,6 +97,20 @@ extensions:
     customField2: customValue2
 {{< /code-block >}}
 
+## Changing service color
+
+Service color is used in trace visualizations. Click the service type icon to change it.
+
+{{< img src="tracing/service_catalog/change_service_color.png" alt="Click the service icon to select a different icon color." style="width:80%;" >}}
+
+## Manage service-related workflows
+[Workflow Automation][14] allows you to automate end-to-end processes across your teams. It integrates with Datadog's Service Catalog to enable dynamic and self-service workflows.
+
+### Updating service type and language
+With [Service Catalog metadata schema 2.2][5], you can specify the type and language for user-defined services or overwrite the auto-detected type and language for instrumented services. Correctly label the service type and language to help other teams further understand what your services do and how to interact with them. 
```

Comment:
Just want to confirm that you want "Service Catalog metadata schema 2.2" to link to the Unified Service Tagging doc.